### DISCUSSION TITLE
port 80 for custom hub and https

### DISCUSSION THREAD

**Discussion by bemyevent** [mozilla/hubs#3164] [Hubs Cloud Questions] [answered]
hi there , We have a problem to deploy hub for one of our customer , there are some problem with port 80 and https and they can't open this port for security reason . Is someone here have a solution to avoid this issue ? I try to change the value in security group but still doesn't work for my customer . thx

**Comment by robinkwilson** [comment, marked as answer]
Duplicate question: See https://github.com/mozilla/hubs/discussions/3091 and https://github.com/mozilla/hubs/discussions/3043
